### PR TITLE
Milestone 2: Components

### PR DESCRIPTION
**Pull request by cliftondavies**
### Completed:
- [x] Gitflow
- [x] ESLint
- [x] Stylelint
- [x] Components directory
- [x] App component
- [x] Display component
- [x] Button component
- [x] Button Panel component
- [x] Ugly page full of non-functional buttons
- [x] Heroku deploy

### CODE REVIEW

**Review by trekab — COMMENTED** [review]
### Invalid! :stop_sign: 
Hi @cliftondavies! Unfortunately, I have to mark your code review request as invalid because your implementation of linting has some errors (__Hint:__ `workflows` NOT `worklows`). Please make sure that the linter checks are running and passing before you request for a code review.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by cliftondavies** [comment]
Thanks for the review @trekab :+1:

The typo has been corrected and both linter checks are running and green. Good spot!

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    </div>
+  );
+
+  return (
+    <div>
+      {buttonGroup('AC', '+/-', '%', '/')}
+      {buttonGroup('7', '8', '9', 'X')}
+      {buttonGroup('4', '5', '6', '-')}
+      {buttonGroup('1', '2', '3', '+')}
+      {buttonGroup('0', '.', '=', '')}
```
Comment (Torres-ssf) [inline src/components/ButtonPanel.js]: - [ ] We can remove the empty value `''` inside the last button group. The last row should have only 3 buttons.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @cliftondavies :hand: 

- [ ] We need to delete the `App.js` that is out of the components directory. Since it was generated by `create-react-app`, we are not using it in our app. We can also delete the `App.test.js` since it's only a boilerplate test.

Happy coding :computer: 
**Sergio Torres**

**Comment by cliftondavies** [comment]
Hi @Torres-ssf :wave:

All requested changes have been applied. See last two commits for reference.

Happy coding!

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by cliftondavies** [comment]
Thanks, @Spayco! 👍

**Pull request merged**